tokio/examples/echo-udp.rs

```diff
@@ -10,44 +10,47 @@
 //!
 //! Each line you type in to the `nc` terminal should be echo'd back to you!
 
-#![deny(warnings, rust_2018_idioms)]
+#![feature(async_await)]
+// #![deny(warnings, rust_2018_idioms)]
```

Comment:
Any particular reason why you commented out this `deny`?

Reply:
Yes, I was playing around. I'll put it back :-)